### PR TITLE
Fix crash with malformed encoded pixel streams, e.g. JPEG quality > 100

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: .gitsubprojects [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # -*- mode: cmake -*-
-git_subproject(ZeroEQ https://github.com/HBPVIS/ZeroEQ.git 1949f24)
+git_subproject(ZeroEQ https://github.com/HBPVIS/ZeroEQ.git ff94fa8)
```
Comment (rdumusc) [inline .gitsubprojects]: Why changed? Please revert, this would break the RestServer without the change for while(receive)

Comment (rdumusc) [inline .gitsubprojects]: or use c63536d maximum (Fix HTTP server response to CORS requests)

Comment (tribal-tec) [inline .gitsubprojects]: Without this, the build is not passing on RHEL due to zmq module upgrade.

Comment (rdumusc) [inline .gitsubprojects]: That's annoying... Then you need to do this too: https://github.com/BlueBrain/Tide/pull/167/files#diff-54d15a4c272c45dd76aad2b4962b05d2R88

Comment (tribal-tec) [inline .gitsubprojects]: Done

**Pull request merged**